### DISCUSSION TITLE
Different reviewers for different target repos

### DISCUSSION THREAD

**Discussion by ataylormays-duetto** [BetaHuhn/repo-file-sync-action#347] [Ideas]
I'm syncing a file from `repoA` to `repoB` and `repoC` and I want those two repos to have separate reviewers, but there's no way to do that with the action. Since the reviewers are specified on the action, the same reviewers are assigned on each target repo. 

A couple implementation ideas:
* If the `sync.yml` allowed us to specify reviewers then this would be possible
* Alternatively allowing a mapping of reviewer->repo in the action would also work

**Comment by shaharkedar** [comment]
I created this PR to address this request:
https://github.com/BetaHuhn/repo-file-sync-action/pull/352
